lua/telescope/builtin/lsp.lua

```diff
@@ -13,21 +13,18 @@ local utils = require "telescope.utils"
 local lsp = {}
 
 lsp.references = function(opts)
-  local params = vim.lsp.util.make_position_params()
+  local params = vim.lsp.util.make_position_params(opts.winnr)
   params.context = { includeDeclaration = true }
 
-  vim.lsp.buf_request(0, "textDocument/references", params, function(err, result, ctx, _config)
+  vim.lsp.buf_request(opts.bufnr, "textDocument/references", params, function(err, result, _, _)
     if err then
       vim.api.nvim_err_writeln("Error when finding references: " .. err.message)
       return
     end
 
     local locations = {}
     if result then
-      vim.list_extend(
-        locations,
-        vim.lsp.util.locations_to_items(result, vim.lsp.get_client_by_id(ctx.client_id).offset_encoding) or {}
-      )
+      vim.list_extend(locations, vim.lsp.util.locations_to_items(result) or {})
```

Comment:
no we need `vim.lsp.get_client_by_id(ctx.client_id).offset_encoding` as second param to `locations_to_items` because of changes in nightly